sys/include/net/gnrc/pktbuf.h

```diff
@@ -225,6 +225,23 @@ gnrc_pktsnip_t *gnrc_pktbuf_remove_snip(gnrc_pktsnip_t *pkt, gnrc_pktsnip_t *sni
  */
 gnrc_pktsnip_t *gnrc_pktbuf_replace_snip(gnrc_pktsnip_t *pkt, gnrc_pktsnip_t *old, gnrc_pktsnip_t *add);
 
+/**
+ * @brief   Reverses snip order of a packet in a write-protected manner.
+ *
+ * This can be used to change the send/receive order of a packet (see
+ * @ref gnrc_pktsnip_t)
+ *
+ * @note    @p pkt is released on failure.
```

Comment:
If the documentation for gnrc_ipv6_ext_demux() puts this information into the @return portion, I'd keep that pattern up for consistency and move this note down as well

Reply:
This change belongs to #10226.

Reply:
But will fix there (won't show up here, until rebased).

Reply:
👍